From 70fc70f9c59f998f7e3132b9b8891acc759a3021 Mon Sep 17 00:00:00 2001
From: Peter O'Connor <peter@solus-project.com>
Date: Tue, 27 Jun 2017 16:15:48 +1000
Subject: [PATCH] Use snappier animations

The 20 animation speed is too slow, and the default could be a little faster

Signed-off-by: Peter O'Connor <peter@solus-project.com>
---
 options.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/options.cpp b/options.cpp
index 2ba1fb1..e4e52d1 100644
--- a/options.cpp
+++ b/options.cpp
@@ -1127,7 +1127,7 @@ Options::MouseCommand Options::wheelToMouseCommand(MouseWheelCommand com, int de

 double Options::animationTimeFactor() const
 {
-    const double factors[] = { 0, 0.2, 0.5, 1, 2, 4, 20 };
+    const double factors[] = { 0, 0.2, 0.5, 0.75, 1, 2, 4 };
     return factors[ animationSpeed ];
 }

--
2.13.2
